crates/cxx-qt-lib/src/gui/qguiapplication.rs

```diff
@@ -67,6 +69,12 @@ mod ffi {
         #[rust_name = "qguiapplication_set_application_version"]
         fn qapplicationSetApplicationVersion(app: Pin<&mut QGuiApplication>, version: &QString);
         #[doc(hidden)]
+        #[rust_name = "qguiapplication_set_font"]
```

Comment:
if these aren't templated anymore can we have direct CXX bindings rather than a C++ trampoline ?

eg

```rust
fn font(self: &QGuiApplication) -> QFont;
fn setFont(self: Pin<&mut QGuiApplication>, font: &QFont);

Reply:
(won't work because these are static methods)